jobs_test.go

```diff
@@ -33,33 +34,45 @@ var (
 	cfg *config.Config
 )
 
-func _inittests(t *testing.T, memoize bool) *Job {
+func _initconfig(t *testing.T) {
+	var err error
+	cfg, err = configurate.Init("test/test_config.yaml")
+	if err != nil {
+		t.Fatal(err)
+	}
+	cfg.Set("condor.run_on_nfs", true)
+	cfg.Set("irods.base", "/path/to/irodsbase")
+	cfg.Set("irods.host", "hostname")
+	cfg.Set("irods.port", "1247")
+	cfg.Set("irods.user", "user")
+	cfg.Set("irods.pass", "pass")
+	cfg.Set("irods.zone", "test")
+	cfg.Set("irods.resc", "")
+	cfg.Set("condor.log_path", "/path/to/logs")
+	cfg.Set("condor.porklock_tag", "test")
+	cfg.Set("condor.filter_files", "foo,bar,baz,blippy")
+	cfg.Set("condor.request_disk", "0")
+}
+
+func inittestsFile(t *testing.T, filename string) *Job {
 	var err error
+	if cfg == nil {
+		_initconfig(t)
+	}
```

Comment:
I didn't see any place where the test configuration was being modified, so I'm only initializing the configuration if it hasn't been initialized already. Again, if someone else could verify that I didn't miss any potential problems, this would be helpful.